### ISSUE TITLE
Can't login to local website after running command 'blt setup'. I don't know username or password

### ISSUE THREAD

**Issue by jezzaKratos** [acquia/blt#2425]
**_My system information:**
* **Operating system type**_: macOS High Sierra
* **Operating system version**: 10.13.2
* **BLT version:** 8.9.11

**Output of `blt doctor**`:
```
Drupal VM was detected. Running blt doctor inside of VM...
Executing command cd /var/www/WebRedesign/docroot/.. && /var/www/WebRedesign/docroot/../vendor/bin/drush cc drush && /var/www/WebRedesign/docroot/../vendor/bin/drush --include=/var/www/WebRedesign/docroot/../vendor/acquia/blt/drush blt-doctor -r /var/www/WebRedesign/docroot/../docroot inside of Drupal VM...
'drush' cache was cleared.                                                                                                                                   [success]
+----------------------+----------------------------------------------------------------------------------+
| Property             | Value                                                                            |
+----------------------+----------------------------------------------------------------------------------+
| %paths               |                                                                                  |
|  - %root             | /var/www/WebRedesign/docroot                                                     |
|  - %site             | sites/default                                                                    |
|  - %modules          | modules                                                                          |
|  - %themes           | themes                                                                           |
|  - %config-sync      | /var/www/WebRedesign/config/default                                              |
|  - %files            | sites/default/files                                                              |
|  - %private          | /var/www/WebRedesign/files-private                                               |
|  - %temp             | /tmp                                                                             |
| admin-theme          | seven                                                                            |
| blt-version          | 8.9.11                                                                           |
| bootstrap            | Successful                                                                       |
| config-sync          | /var/www/WebRedesign/config/default                                              |
| db-driver            | mysql                                                                            |
| db-hostname          | localhost                                                                        |
| db-name              | drupal                                                                           |
| db-password          | drupal                                                                           |
| db-port              | 3306                                                                             |
| db-status            | Connected                                                                        |
| db-username          | drupal                                                                           |
| dev-desktop-enabled  | false                                                                            |
| drupal-settings-file | sites/default/settings.php                                                       |
| drupal-version       | 8.4.3                                                                            |
| drupal-vm-enabled    | true                                                                             |
| drush-alias-files    | /var/www/WebRedesign/docroot/../drush/site-aliases/example.local.aliases.drushrc |
|                      | .php                                                                             |
|                      | /var/www/WebRedesign/docroot/../drush/site-aliases/aliases.drushrc.php           |
|                      | /var/www/WebRedesign/docroot/../drush/site-aliases/example.acsf.aliases.drushrc. |
|                      | php                                                                              |
| drush-conf           | /var/www/WebRedesign/drush/drushrc.php                                           |
| drush-script         | /var/www/WebRedesign/vendor/drush/drush/drush.php                                |
| drush-temp           | /tmp                                                                             |
| drush-version        | 8.1.15                                                                           |
| files                | sites/default/files                                                              |
| install-profile      | lightning                                                                        |
| local-drushrc        | /var/www/WebRedesign/docroot/sites/default/local.drushrc.php                     |
| local-settings       | /var/www/WebRedesign/docroot/sites/default/settings/local.settings.php           |
| modules              | modules                                                                          |
| php-bin              | /usr/bin/php                                                                     |
| php-conf             | /etc/php/5.6/cli/php.ini                                                         |
| php-mysql            | /var/run/mysqld/mysqld.sock                                                      |
| php-os               | Linux                                                                            |
| private              | /var/www/WebRedesign/files-private                                               |
| root                 | /var/www/WebRedesign/docroot                                                     |
| shell                |                                                                                  |
| site                 | sites/default                                                                    |
| temp                 | /tmp                                                                             |
| theme                | stark                                                                            |
| themes               | themes                                                                           |
| uri                  | http://local.WebRedesign.com                                                     |
| user                 |                                                                                  |
+----------------------+----------------------------------------------------------------------------------+
+--------------------------+-------------------------------------------------------------+
| Check                    | Outcome                                                     |
+--------------------------+-------------------------------------------------------------+
| checkDocrootExists       | Found docroot.                                              |
| checkCoreExists          | Drupal core exists                                          |
| checkSettingsFile        | BLT settings are included in settings file.                 |
| checkLocalSettingsFile   | Found your local settings file.                             |
| checkLocalDrushFile      | Found your local drush settings file.                       |
| checkUri                 | $options['uri'] is set.                                     |
| checkUriResponse         | Received a response from site http://local.WebRedesign.com. |
| checkFileSystem:%files   | Public files directory is writable.                         |
| checkFileSystem:%private | Private files directory is writable.                        |
| checkFileSystem:%temp    | Temporary files directory is writable.                      |
| checkDbConnection        | Connected to database.                                      |
| checkDrupalBootstrapped  | Bootstrapped Drupal via drush.                              |
| checkDrupalInstalled     | Drupal is installed.                                        |
| checkCaching:page        | Drupal cache is disabled.                                   |
| checkCaching:css         | CSS preprocessing is disabled.                              |
| checkCaching:js          | JS preprocessing is disabled.                               |
| checkCiConfig            | Git remotes are set in project.yml.                         |
| checkComposer:require    | acquia/blt is in composer.json's require object.            |
| checkBehat:exists        | Behat local settings file exists.                           |
| checkBehat:root          | Behat drupal_root is set correctly for Drupal VM.           |
| checkProjectYml:keys     | project.yml has no deprecated keys.                         |
| checkDrupalVm:alias      | drush.aliases.local exists your drush aliases file.         |
| checkDrupalVm            | Drupal VM is configured correctly.                          |
| checkPhpDateTimezone     | PHP setting for date.timezone is correctly set              |
+--------------------------+-------------------------------------------------------------+
Connection to 127.0.0.1 closed.
```

**When I run this command:**
```
blt setup
```

**I get the following output:**
```
$ blt setup
Setting up local environment for site 'default' using drush alias @WebRedesign.local
> setup:build
> setup:behat
Executing command blt 'setup:behat' --define=drush.alias=self inside of Drupal VM...
[warning] The xDebug extension is loaded. This will significantly decrease performance.
Connection to 127.0.0.1 closed.
> setup:composer:install
Gathering patches for root package.
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
> setup:git-hooks
Installing pre-commit git hook...
Installing commit-msg git hook...
> setup:settings
> frontend
Executing command blt frontend --define=drush.alias=self inside of Drupal VM...
[warning] The xDebug extension is loaded. This will significantly decrease performance.
> frontend:setup
Executing frontend-setup target hook...
[ExecStack] ./scripts/frontend/install-node.sh 6.12.0
[ExecStack] Running ./scripts/frontend/install-node.sh 6.12.0 in /var/www/WebRedesign
Please run the following command:
source ~/.bashrc && nvm use --delete-prefix 6.12.0
[ExecStack] Done in 0.061s
> frontend:build
Executing frontend-build target hook...
[ExecStack] ./scripts/frontend/compile-sass.sh
[ExecStack] Running ./scripts/frontend/compile-sass.sh in /var/www/WebRedesign
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN checkPermissions Missing write access to /home/vagrant/.npm-global/lib/node_modules
/home/vagrant/.npm-global/lib
`-- gulp@3.9.1
  +-- archy@1.0.0
  +-- chalk@1.1.3
  | +-- ansi-styles@2.2.1
  | +-- escape-string-regexp@1.0.5
  | +-- has-ansi@2.0.0
  | | `-- ansi-regex@2.1.1
  | +-- strip-ansi@3.0.1
  | `-- supports-color@2.0.0
  +-- deprecated@0.0.1
  +-- gulp-util@3.0.8
  | +-- array-differ@1.0.0
  | +-- array-uniq@1.0.3
  | +-- beeper@1.1.1
  | +-- dateformat@2.2.0
  | +-- fancy-log@1.3.2
  | | +-- ansi-gray@0.1.1
  | | | `-- ansi-wrap@0.1.0
  | | +-- color-support@1.1.3
  | | `-- time-stamp@1.1.0
  | +-- gulplog@1.0.0
  | | `-- glogg@1.0.0
  | +-- has-gulplog@0.1.0
  | | `-- sparkles@1.0.0
  | +-- lodash._reescape@3.0.0
  | +-- lodash._reevaluate@3.0.0
  | +-- lodash._reinterpolate@3.0.0
  | +-- lodash.template@3.6.2
  | | +-- lodash._basecopy@3.0.1
  | | +-- lodash._basetostring@3.0.1
  | | +-- lodash._basevalues@3.0.0
  | | +-- lodash._isiterateecall@3.0.9
  | | +-- lodash.escape@3.2.0
  | | | `-- lodash._root@3.0.1
  | | +-- lodash.keys@3.1.2
  | | | +-- lodash._getnative@3.9.1
  | | | +-- lodash.isarguments@3.1.0
  | | | `-- lodash.isarray@3.0.4
  | | +-- lodash.restparam@3.6.1
  | | `-- lodash.templatesettings@3.1.1
  | +-- multipipe@0.1.2
  | | `-- duplexer2@0.0.2
  | |   `-- readable-stream@1.1.14
  | +-- object-assign@3.0.0
  | +-- replace-ext@0.0.1
  | +-- through2@2.0.3
  | | +-- readable-stream@2.3.3
  | | | +-- core-util-is@1.0.2
  | | | +-- inherits@2.0.3
  | | | +-- isarray@1.0.0
  | | | +-- process-nextick-args@1.0.7
  | | | +-- safe-buffer@5.1.1
  | | | +-- string_decoder@1.0.3
  | | | `-- util-deprecate@1.0.2
  | | `-- xtend@4.0.1
  | `-- vinyl@0.5.3
  |   +-- clone@1.0.3
  |   `-- clone-stats@0.0.1
  +-- interpret@1.1.0
  +-- liftoff@2.5.0
  | +-- extend@3.0.1
  | +-- findup-sync@2.0.0
  | | +-- detect-file@1.0.0
  | | +-- is-glob@3.1.0
  | | | `-- is-extglob@2.1.1
  | | +-- micromatch@3.1.5
  | | | +-- arr-diff@4.0.0
  | | | +-- array-unique@0.3.2
  | | | +-- braces@2.3.0
  | | | | +-- arr-flatten@1.1.0
  | | | | +-- fill-range@4.0.0
  | | | | | +-- is-number@3.0.0
  | | | | | | `-- kind-of@3.2.2
  | | | | | +-- repeat-string@1.6.1
  | | | | | `-- to-regex-range@2.1.1
  | | | | +-- repeat-element@1.1.2
  | | | | +-- snapdragon-node@2.1.1
  | | | | | `-- snapdragon-util@3.0.1
  | | | | |   `-- kind-of@3.2.2
  | | | | `-- split-string@3.1.0
  | | | |   `-- extend-shallow@3.0.2
  | | | |     +-- assign-symbols@1.0.0
  | | | |     `-- is-extendable@1.0.1
  | | | +-- define-property@1.0.0
  | | | | `-- is-descriptor@1.0.2
  | | | |   +-- is-accessor-descriptor@1.0.0
  | | | |   `-- is-data-descriptor@1.0.0
  | | | +-- extend-shallow@2.0.1
  | | | | `-- is-extendable@0.1.1
  | | | +-- extglob@2.0.3
  | | | | `-- expand-brackets@2.1.4
  | | | |   +-- define-property@0.2.5
  | | | |   | `-- is-descriptor@0.1.6
  | | | |   |   +-- is-accessor-descriptor@0.1.6
  | | | |   |   | `-- kind-of@3.2.2
  | | | |   |   +-- is-data-descriptor@0.1.4
  | | | |   |   | `-- kind-of@3.2.2
  | | | |   |   `-- kind-of@5.1.0
  | | | |   `-- posix-character-classes@0.1.1
  | | | +-- fragment-cache@0.2.1
  | | | +-- kind-of@6.0.2
  | | | +-- nanomatch@1.2.7
  | | | | +-- is-odd@1.0.0
  | | | | `-- kind-of@5.1.0
  | | | +-- regex-not@1.0.0
  | | | +-- snapdragon@0.8.1
  | | | | +-- base@0.11.2
  | | | | | +-- cache-base@1.0.1
  | | | | | | +-- collection-visit@1.0.0
  | | | | | | | +-- map-visit@1.0.0
  | | | | | | | `-- object-visit@1.0.1
  | | | | | | +-- get-value@2.0.6
  | | | | | | +-- has-value@1.0.0
  | | | | | | | `-- has-values@1.0.0
  | | | | | | |   `-- kind-of@4.0.0
  | | | | | | +-- set-value@2.0.0
  | | | | | | +-- to-object-path@0.3.0
  | | | | | | | `-- kind-of@3.2.2
  | | | | | | +-- union-value@1.0.0
  | | | | | | | `-- set-value@0.4.3
  | | | | | | `-- unset-value@1.0.0
  | | | | | |   `-- has-value@0.3.1
  | | | | | |     +-- has-values@0.1.4
  | | | | | |     `-- isobject@2.1.0
  | | | | | |       `-- isarray@1.0.0
  | | | | | +-- class-utils@0.3.5
  | | | | | | +-- arr-union@3.1.0
  | | | | | | +-- define-property@0.2.5
  | | | | | | | `-- is-descriptor@0.1.6
  | | | | | | |   +-- is-accessor-descriptor@0.1.6
  | | | | | | |   | `-- kind-of@3.2.2
  | | | | | | |   +-- is-data-descriptor@0.1.4
  | | | | | | |   | `-- kind-of@3.2.2
  | | | | | | |   `-- kind-of@5.1.0
  | | | | | | `-- static-extend@0.1.2
  | | | | | |   +-- define-property@0.2.5
  | | | | | |   | `-- is-descriptor@0.1.6
  | | | | | |   |   +-- is-accessor-descriptor@0.1.6
  | | | | | |   |   | `-- kind-of@3.2.2
  | | | | | |   |   +-- is-data-descriptor@0.1.4
  | | | | | |   |   | `-- kind-of@3.2.2
  | | | | | |   |   `-- kind-of@5.1.0
  | | | | | |   `-- object-copy@0.1.0
  | | | | | |     +-- copy-descriptor@0.1.1
  | | | | | |     +-- define-property@0.2.5
  | | | | | |     | `-- is-descriptor@0.1.6
  | | | | | |     |   +-- is-accessor-descriptor@0.1.6
  | | | | | |     |   +-- is-data-descriptor@0.1.4
  | | | | | |     |   `-- kind-of@5.1.0
  | | | | | |     `-- kind-of@3.2.2
  | | | | | +-- component-emitter@1.2.1
  | | | | | +-- mixin-deep@1.3.0
  | | | | | | `-- is-extendable@1.0.1
  | | | | | `-- pascalcase@0.1.1
  | | | | +-- debug@2.6.9
  | | | | | `-- ms@2.0.0
  | | | | +-- define-property@0.2.5
  | | | | | `-- is-descriptor@0.1.6
  | | | | |   +-- is-accessor-descriptor@0.1.6
  | | | | |   | `-- kind-of@3.2.2
  | | | | |   +-- is-data-descriptor@0.1.4
  | | | | |   | `-- kind-of@3.2.2
  | | | | |   `-- kind-of@5.1.0
  | | | | +-- source-map@0.5.7
  | | | | +-- source-map-resolve@0.5.1
  | | | | | +-- atob@2.0.3
  | | | | | +-- decode-uri-component@0.2.0
  | | | | | +-- resolve-url@0.2.1
  | | | | | +-- source-map-url@0.4.0
  | | | | | `-- urix@0.1.0
  | | | | `-- use@2.0.2
  | | | |   +-- define-property@0.2.5
  | | | |   | `-- is-descriptor@0.1.6
  | | | |   |   +-- is-accessor-descriptor@0.1.6
  | | | |   |   | `-- kind-of@3.2.2
  | | | |   |   +-- is-data-descriptor@0.1.4
  | | | |   |   | `-- kind-of@3.2.2
  | | | |   |   `-- kind-of@5.1.0
  | | | |   `-- lazy-cache@2.0.2
  | | | |     `-- set-getter@0.1.0
  | | | `-- to-regex@3.0.1
  | | |   `-- define-property@0.2.5
  | | |     `-- is-descriptor@0.1.6
  | | |       +-- is-accessor-descriptor@0.1.6
  | | |       | `-- kind-of@3.2.2
  | | |       +-- is-data-descriptor@0.1.4
  | | |       | `-- kind-of@3.2.2
  | | |       `-- kind-of@5.1.0
  | | `-- resolve-dir@1.0.1
  | |   `-- global-modules@1.0.0
  | |     +-- global-prefix@1.0.2
  | |     | +-- ini@1.3.5
  | |     | `-- which@1.3.0
  | |     |   `-- isexe@2.0.0
  | |     `-- is-windows@1.0.1
  | +-- fined@1.1.0
  | | +-- expand-tilde@2.0.2
  | | | `-- homedir-polyfill@1.0.1
  | | |   `-- parse-passwd@1.0.0
  | | +-- object.defaults@1.1.0
  | | | +-- array-each@1.0.1
  | | | `-- array-slice@1.1.0
  | | +-- object.pick@1.3.0
  | | `-- parse-filepath@1.0.2
  | |   +-- is-absolute@1.0.0
  | |   | `-- is-relative@1.0.0
  | |   |   `-- is-unc-path@1.0.0
  | |   |     `-- unc-path-regex@0.1.2
  | |   +-- map-cache@0.2.2
  | |   `-- path-root@0.1.1
  | |     `-- path-root-regex@0.1.2
  | +-- flagged-respawn@1.0.0
  | +-- is-plain-object@2.0.4
  | | `-- isobject@3.0.1
  | +-- object.map@1.0.1
  | | +-- for-own@1.0.0
  | | | `-- for-in@1.0.2
  | | `-- make-iterator@1.0.0
  | |   `-- kind-of@3.2.2
  | |     `-- is-buffer@1.1.6
  | +-- rechoir@0.6.2
  | `-- resolve@1.5.0
  |   `-- path-parse@1.0.5
  +-- minimist@1.2.0
  +-- orchestrator@0.3.8
  | +-- end-of-stream@0.1.5
  | | `-- once@1.3.3
  | |   `-- wrappy@1.0.2
  | +-- sequencify@0.0.7
  | `-- stream-consume@0.1.0
  +-- pretty-hrtime@1.0.3
  +-- semver@4.3.6
  +-- tildify@1.2.0
  | `-- os-homedir@1.0.2
  +-- v8flags@2.1.1
  | `-- user-home@1.1.1
  `-- vinyl-fs@0.3.14
    +-- defaults@1.0.3
    +-- glob-stream@3.1.18
    | +-- glob@4.5.3
    | | `-- inflight@1.0.6
    | +-- glob2base@0.0.12
    | | `-- find-index@0.1.1
    | +-- minimatch@2.0.10
    | | `-- brace-expansion@1.1.8
    | |   +-- balanced-match@1.0.0
    | |   `-- concat-map@0.0.1
    | +-- ordered-read-streams@0.1.0
    | +-- through2@0.6.5
    | | `-- readable-stream@1.0.34
    | `-- unique-stream@1.0.0
    +-- glob-watcher@0.0.6
    | `-- gaze@0.5.2
    |   `-- globule@0.1.0
    |     +-- glob@3.1.21
    |     | +-- graceful-fs@1.2.3
    |     | `-- inherits@1.0.2
    |     +-- lodash@1.0.2
    |     `-- minimatch@0.2.14
    |       +-- lru-cache@2.7.3
    |       `-- sigmund@1.0.1
    +-- graceful-fs@3.0.11
    | `-- natives@1.1.1
    +-- mkdirp@0.5.1
    | `-- minimist@0.0.8
    +-- strip-bom@1.0.0
    | +-- first-chunk-stream@1.0.0
    | `-- is-utf8@0.2.1
    +-- through2@0.6.5
    | `-- readable-stream@1.0.34
    |   +-- isarray@0.0.1
    |   `-- string_decoder@0.10.31
    `-- vinyl@0.4.6
      `-- clone@0.2.0

npm ERR! Linux 4.4.0-87-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "--global" "gulp"
npm ERR! node v6.12.3
npm ERR! npm  v3.10.10
npm ERR! path /home/vagrant/.npm-global/lib/node_modules
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access

npm ERR! Error: EACCES: permission denied, access '/home/vagrant/.npm-global/lib/node_modules'
npm ERR!     at Error (native)
npm ERR!  { Error: EACCES: permission denied, access '/home/vagrant/.npm-global/lib/node_modules'
npm ERR!     at Error (native)
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/home/vagrant/.npm-global/lib/node_modules' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /var/www/WebRedesign/npm-debug.log

> VisionAustralia@0.1.0 postinstall /var/www/WebRedesign/docroot/themes/custom/va
> find node_modules/ -name '*.info' -type f -delete

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN eslint-config-airbnb-base@11.3.2 requires a peer of eslint-plugin-import@^2.7.0 but none was installed.
[05:10:04] Using gulpfile /var/www/WebRedesign/docroot/themes/custom/va/gulpfile.js
[05:10:04] Starting 'compile:sass'...
[05:10:05] Starting 'minify:css'...
[05:10:06] Finished 'minify:css' after 806 ms
[05:10:06] Finished 'compile:sass' after 2.12 s
[05:10:06] Starting 'build'...
[05:10:06] Finished 'build' after 22 μs
[05:10:06] Starting 'default'...
[05:10:06] Finished 'default' after 9.94 μs

> VisionAustralia@0.1.0 postinstall /var/www/WebRedesign/docroot/themes/custom/carols
> find node_modules/ -name '*.info' -type f -delete

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN eslint-config-airbnb-base@11.3.2 requires a peer of eslint-plugin-import@^2.7.0 but none was installed.
[05:10:17] Using gulpfile /var/www/WebRedesign/docroot/themes/custom/carols/gulpfile.js
[05:10:17] Starting 'compile:sass'...
[05:10:17] Starting 'minify:css'...
[05:10:18] Finished 'minify:css' after 602 ms
[05:10:18] Finished 'compile:sass' after 1.58 s
[05:10:18] Starting 'build'...
[05:10:18] Finished 'build' after 25 μs
[05:10:18] Starting 'default'...
[05:10:18] Finished 'default' after 8.39 μs

> VisionAustralia@0.1.0 postinstall /var/www/WebRedesign/docroot/themes/custom/radio
> find node_modules/ -name '*.info' -type f -delete

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN eslint-config-airbnb-base@11.3.2 requires a peer of eslint-plugin-import@^2.7.0 but none was installed.
[05:10:28] Using gulpfile /var/www/WebRedesign/docroot/themes/custom/radio/gulpfile.js
[05:10:28] Starting 'compile:sass'...
[05:10:28] Starting 'minify:css'...
[05:10:29] Finished 'minify:css' after 620 ms
[05:10:30] Finished 'compile:sass' after 1.58 s
[05:10:30] Starting 'build'...
[05:10:30] Finished 'build' after 23 μs
[05:10:30] Starting 'default'...
[05:10:30] Finished 'default' after 12 μs

> VisionAustralia@0.1.0 postinstall /var/www/WebRedesign/docroot/themes/custom/seda
> find node_modules/ -name '*.info' -type f -delete

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN eslint-config-airbnb-base@11.3.2 requires a peer of eslint-plugin-import@^2.7.0 but none was installed.
[05:10:40] Using gulpfile /var/www/WebRedesign/docroot/themes/custom/seda/gulpfile.js
[05:10:40] Starting 'compile:sass'...
[05:10:40] Starting 'minify:css'...
[05:10:41] Finished 'minify:css' after 605 ms
[05:10:41] Finished 'compile:sass' after 1.58 s
[05:10:41] Starting 'build'...
[05:10:41] Finished 'build' after 23 μs
[05:10:41] Starting 'default'...
[05:10:41] Finished 'default' after 7.96 μs
[ExecStack] Done in 56.606s
Connection to 127.0.0.1 closed.
> setup:hash-salt
Hash salt already exists.
> setup:drupal:install
Executing command blt 'setup:drupal:install' --define=drush.alias=self inside of Drupal VM...
[warning] The xDebug extension is loaded. This will significantly decrease performance.
> internal:drupal:install
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/WebRedesign/vendor/bin/drush site-install lightning install_configure_form.update_status_module='array(FALSE,FALSE)' install_configure_form.enable_update_status_module=NULL --site-name='Web Redesign' --site-mail=no-reply@acquia.com --account-name='+pN7iyTXMI' --account-mail=no-reply@acquia.com --locale=en --config-dir=../config/default --uri=default --yes --verbose in /var/www/WebRedesign/docroot
Configuration import directory ../config/default does not contain any configuration; will skip import.                                                       [warning]
Executing: mysql --defaults-extra-file=/tmp/drush_ruTni8 --database=drupal --host=localhost --port=3306 --silent  < /tmp/drush_dulpWb > /dev/null
You are about to DROP all tables in your 'drupal' database. Do you want to continue? (y/n): y
Sites directory /var/www/WebRedesign/docroot/sites/default already exists - proceeding.                                                                         [notice]
Executing: mysql --defaults-extra-file=/tmp/drush_MvhJ20 --database=drupal --host=localhost --port=3306 --silent  < /tmp/drush_8J2wF4 > /dev/null
Executing: mysql --defaults-extra-file=/tmp/drush_7OZ8OT --database=drupal --host=localhost --port=3306 --silent  < /tmp/drush_RxBUqX
Executing: mysql --defaults-extra-file=/tmp/drush_OZfbEM --database=drupal --host=localhost --port=3306 --silent  < /tmp/drush_NdLBeQ
Starting Drupal installation. This takes a while. Consider using the --notify global option.                                                                 [ok]
exception 'Drupal\Core\Config\PreExistingConfigException' with message 'Configuration objects (core.entity_form_display.media.image.default,                 [error]
core.entity_view_display.media.image.default, media.type.image) provided by lightning_media_image already exist in active configuration' in
/var/www/WebRedesign/docroot/core/lib/Drupal/Core/Config/PreExistingConfigException.php:65
Stack trace:
#0 /var/www/WebRedesign/docroot/core/lib/Drupal/Core/Config/ConfigInstaller.php(492):
Drupal\Core\Config\PreExistingConfigException::create('lightning_media...', Array)
#1 /var/www/WebRedesign/docroot/core/lib/Drupal/Core/ProxyClass/Config/ConfigInstaller.php(132):
Drupal\Core\Config\ConfigInstaller->checkConfigurationToInstall('module', 'lightning_media...')
#2 /var/www/WebRedesign/docroot/core/lib/Drupal/Core/Extension/ModuleInstaller.php(145):
Drupal\Core\ProxyClass\Config\ConfigInstaller->checkConfigurationToInstall('module', 'lightning_media...')
#3 /var/www/WebRedesign/docroot/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php(83): Drupal\Core\Extension\ModuleInstaller->install(Array,
false)
#4 /var/www/WebRedesign/docroot/core/includes/install.core.inc(1813): Drupal\Core\ProxyClass\Extension\ModuleInstaller->install(Array, false)
#5 [internal function]: _install_module_batch('lightning_media...', 'Media Image', Array)
#6 /var/www/WebRedesign/docroot/core/includes/batch.inc(294): call_user_func_array('_install_module...', Array)
#7 /var/www/WebRedesign/docroot/core/includes/form.inc(875): _batch_process()
#8 /var/www/WebRedesign/docroot/core/includes/install.core.inc(634): batch_process(Object(Drupal\Core\Url), Object(Drupal\Core\Url))
#9 /var/www/WebRedesign/docroot/core/includes/install.core.inc(555): install_run_task(Array, Array)
#10 /var/www/WebRedesign/docroot/core/includes/install.core.inc(117): install_run_tasks(Array)
#11 /var/www/WebRedesign/vendor/drush/drush/includes/drush.inc(726): install_drupal(Object(Composer\Autoload\ClassLoader), Array)
#12 /var/www/WebRedesign/vendor/drush/drush/includes/drush.inc(711): drush_call_user_func_array('install_drupal', Array)
#13 /var/www/WebRedesign/vendor/drush/drush/commands/core/drupal/site_install.inc(82): drush_op('install_drupal', Object(Composer\Autoload\ClassLoader),
Array)
#14 /var/www/WebRedesign/vendor/drush/drush/commands/core/site_install.drush.inc(255): drush_core_site_install_version('lightning', Array)
#15 [internal function]: drush_core_site_install('lightning', 'install_configu...', 'install_configu...')
#16 /var/www/WebRedesign/vendor/drush/drush/includes/command.inc(422): call_user_func_array('drush_core_site...', Array)
#17 /var/www/WebRedesign/vendor/drush/drush/includes/command.inc(231): _drush_invoke_hooks(Array, Array)
#18 [internal function]: drush_command('lightning', 'install_configu...', 'install_configu...')
#19 /var/www/WebRedesign/vendor/drush/drush/includes/command.inc(199): call_user_func_array('drush_command', Array)
#20 /var/www/WebRedesign/vendor/drush/drush/lib/Drush/Boot/BaseBoot.php(67): drush_dispatch(Array)
#21 /var/www/WebRedesign/vendor/drush/drush/includes/preflight.inc(66): Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#22 /var/www/WebRedesign/vendor/drush/drush/drush.php(12): drush_main()
#23 {main}
Command dispatch complete                                                                                                                                       [notice]
[Acquia\Blt\Robo\Tasks\DrushTask]  Exit code 1  Time 01:41
[error]  Failed to install Drupal!
[error]  Command `internal:drupal:install ` exited with code 1.
Connection to 127.0.0.1 closed.
> setup:toggle-modules
Executing command blt 'setup:toggle-modules' --define=drush.alias=self inside of Drupal VM...
[warning] The xDebug extension is loaded. This will significantly decrease performance.
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/WebRedesign/vendor/bin/drush pm-enable dblog devel seckit views_ui --uri=default --yes in /var/www/WebRedesign/docroot
dblog is already enabled.                                                                                                                                    [ok]
views_ui is already enabled.                                                                                                                                 [ok]
The following extensions will be enabled: devel, seckit
Do you really want to continue? (y/n): y
devel was enabled successfully.                                                                                                                              [ok]
devel defines the following permissions: access devel information, execute php code, switch users
seckit was enabled successfully.                                                                                                                             [ok]
seckit defines the following permissions: administer seckit
[Acquia\Blt\Robo\Tasks\DrushTask] Done in 7.785s
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/WebRedesign/vendor/bin/drush pm-uninstall acsf acquia_connector shield --uri=default --yes in /var/www/WebRedesign/docroot
Extension acsf was not found and will not be uninstalled.                                                                                                    [warning]
acquia_connector is already uninstalled.                                                                                                                     [ok]
shield is already uninstalled.                                                                                                                               [ok]
There were no extensions that could be uninstalled.                                                                                                          [ok]
[Acquia\Blt\Robo\Tasks\DrushTask] Done in 0.943s
Connection to 127.0.0.1 closed.
> install-alias
The BLT alias is already installed and up to date.
$
```

**And I expected this to happen:**
I've read that if you run the command 'blt setup', a randrom username and password is generated and this is what you use to login when you load the localsite on a browser, but I didn't see it. 

I have access to the site by url: local.[projectName].com, however, when I load the site on a browser, it's showing the drupal username and password screen prompt. I don't have the username or password to login to this. Where is this located in the code folder?

**Comment by grasmash** [comment]
In the output you pasted, Drupal failed to install due to a configuration issue. You’ll want to resolve that first.

I’d recommend using ‘drush uli’ to login locally.

**Comment by jezzaKratos** [comment]
In the 'blt doctor' output, it has, "...| checkDrupalInstalled     | Drupal is installed. ..."

Drupal is already installed, could that why it failed to install Drupal (again)? 

I tried 'drush uli', but got this output, "We could not find an applicable site for that command. [error]"

**Comment by jezzaKratos** [comment]
I tried 'drush sa' and the output was, '@none'

**Comment by grasmash** [comment]
Hm, that sounds like a bigger issue. You need drush to be working before BLT can work.

What happens when you run `drush si lightning --config-dir=../config/default` ? Try debugging that directly.

**Comment by jezzaKratos** [comment]
I executed that command, $ drush si lightning --config-dir=../config/default

and this is the output...
```
Command site-install needs a higher bootstrap level to run - you will need to invoke drush from a more functional     [error]
Drupal environment to run this command.
The drush command 'si lightning' could not be executed.                                                               [error]

```

Just in case, FYI. These are the steps I followed in my attempt to setup Acquia local environment on my machine: 

Installed the following: Git, Composer, PHP5.6+, Vagrant 5.1+, Virtualbox 1.8.6+
1.            Checkout the source code from Bitbucket.
2.            Move into Code folder.
4.            Run command from code folder: composer update, composer install
5.            Install blt alias: composer run-script blt-alias
6.            Run command: blt vm
7.            Run command: blt setup
8.            Run command: blt doctor [there are no red highlighted issues as per above.
